src/inference/src/os/lin/lin_system_conf.cpp

```diff
@@ -547,7 +547,8 @@ void parse_cache_info_linux(const std::vector<std::vector<std::string>> system_i
                         sub_str = system_info_table[n][info_index].substr(endpos + 1);
                         core_2 = std::stoi(sub_str);
 
-                        if ((info_index == 1) && (core_2 - core_1 == 1)) {
+                        if ((info_index == 1) && (core_2 - core_1 == 1) &&
```

Comment:
Is it possible to add some unit test which catch this core-dump issue on E3950?
OR updated test `LinuxCpuMapCacheParserTests` will cover it?

Reply:
New test case for E3950 had been added in `LinuxCpuMapCacheParserTests`